test/functional/kernel/region/test-kernel-region-openmp.cpp

```diff
@@ -0,0 +1,54 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "tests/test-kernel-region.hpp"
+
+#include "../../forall/test-forall-utils.hpp"
+
+using SequentialKernelRegionExecPols = 
+  camp::list< 
+
+    RAJA::KernelPolicy<
+      RAJA::statement::Region<RAJA::omp_parallel_region,
+        RAJA::statement::For<0, RAJA::omp_for_nowait_exec,
+          RAJA::statement::Lambda<0>
+        >,
+        RAJA::statement::For<0, RAJA::omp_for_nowait_exec,
+          RAJA::statement::Lambda<1>
+        >,
```

Comment:
Should there be a sync threads here?

Reply:
@artv3 I added a test for openmp sync in kernel region tests.  Please review. Thanks.

Reply:
@artv3 are you good with the sync test I added?

Reply:
Yup look good!